lib/xcmv/group_membership.rb

```diff
@@ -1,18 +1,35 @@
+class Xcodeproj::Project::Object::PBXGroup
+  # Returns an array of targets that have build files in `group`.
+  def sibling_targets
+    siblings = children.to_set
+    compiled_targets = project.targets.select{ |t| t.respond_to?(:source_build_phase) }
+    compiled_targets.select{ |t| t.source_build_phase.files_references.any?{ |f| siblings.include?(f) } }
+  end
+end
 
 module XcodeMove
   class GroupMembership
+    attr_reader :group, :parent, :project
     def initialize(group)
       @group = group
       @project = group.project
       @siblings = @group.children.to_set
+      @parent = @group == @project.main_group ? nil : @group.parent
```

Comment:
Since `sibling_targets` belongs to `PBXGroup` now, I don't actually think this is necessary anymore.